src/containers/Navbar/AccountMenu.js

```diff
@@ -52,6 +56,7 @@ function AccountMenu() {
         transformOrigin={{ vertical: 'top', horizontal: 'right' }}
         open={Boolean(anchorEl)}
         onClose={closeAccountMenu}>
+        <MenuItem onClick={goToMyRequests}>My Requests</MenuItem>
```

Comment:
FYI, we can use `component={Link} to={MY_REQUESTS_PATH} onClick={closeAccountMenu}`to have the menu entry act like an href.  This way, if desired, users can Cmd+Click to open in new tab, etc.  Not a big deal, but would be good to use going forward.

Reply:
Agreed - I like doing this whenever possible,